cnv/cnv_release_notes/cnv-release-notes.adoc

```diff
@@ -114,6 +114,11 @@ $ export TARGET_NAMESPACE=openshift-cnv CNV_CHANNEL=2.3 && oc patch -n "${TARGET
 +
 This command points your subscription to upgrade channel `2.3` and enables automatic updates.
 
+* The Containerized Data Importer (CDI) does not always use the `scratchSpaceStorageClass` setting in the CDIConfig object for importing and uploading operations.
+Instead, the CDI uses the default storage class to allocate scratch space.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1828198[*BZ#1828198*])
++
+As a workaround, ensure you have defined a xref:../../storage/dynamic-provisioning.adoc#storage-class-annotations_dynamic-provisioning[default storage class] for your cluster.
```

Comment:
I'm not so sure about where the link should point to.
The way to set a default storage class:
oc patch storageclass STORAGE_CLASS_NAME -p '{"metadata": {"annotations": {"storageclass.kubernetes.io/is-default-class": "true"}}}'

Reply:
Updated as suggest